tsconfig.build.json

```diff
@@ -1,9 +1,13 @@
 {
-	"extends": "./config/tsconfig.base",
-	"exclude": [
-		"node_modules/**",
-		"src/**/*spec.ts",
-		"src/**/__tests__/*",
-		"src/**/__mocks__/*"
-	]
-}
\ No newline at end of file
+	"extends": "@sofie-automation/code-standard-preset/ts/tsconfig.lib",
+	"exclude": ["node_modules/**", "src/**/*spec.ts", "src/**/__tests__/*", "src/**/__mocks__/*"],
```

Comment:
could the excludes perhaps be moved up into the preset-repo?

Reply:
could... probably? cc @Julusian 
should... that's a different question I guess. it would imply an enforced file structure, which is not something we enforce today (more or less assumed but not enforced)

Reply:
The includes/excludes are all relative to the config file path, but I guess it should work though.
I think having it in this repo is fine though.
Also I am about to suggest another change which will cause a conflict with pushing it upstream too